RealmLoginKit Apple/RealmLoginKit/Controllers/LoginViewController.swift

```diff
@@ -595,17 +595,16 @@ public class LoginViewController: UIViewController, UITableViewDataSource, UITab
         
         saveLoginCredentials()
         
+        var scheme: String?
         var formattedURL = serverURL
         if let schemeRange = formattedURL?.range(of: "://") {
+            scheme = formattedURL?.substring(to: schemeRange.lowerBound)
             formattedURL = formattedURL?.substring(from: schemeRange.upperBound)
         }
         
-        if formattedURL?.range(of: ":") == nil {
-            formattedURL = "\(formattedURL!):9080"
-        }
-        
         let credentials = RLMSyncCredentials(username: email!, password: password!, register: isRegistering)
-        RLMSyncUser.__logIn(with: credentials, authServerURL: URL(string: "http://\(formattedURL!)")!, timeout: 30, onCompletion: { (user, error) in
+        let authScheme = scheme == "realms" ? "https" : "http"
+        RLMSyncUser.__logIn(with: credentials, authServerURL: URL(string: "\(authScheme)://\(formattedURL!)")!, timeout: 30, onCompletion: { (user, error) in
```

Comment:
Is there a better API we can use for this? Keeping in mind this framework is only importing Realm Objective-C and not Realm Swift.